integration-tests/jest-reporter.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash/fp')
+const { parse } = require('path')
+const { createWriteStream, existsSync, mkdirSync } = require('fs')
+
+
+module.exports = class JestReporter {
+  constructor(_globalConfig, _options) {
+    this.logRootDir = process.env.LOG_DIR || '/tmp/test-results'
+  }
+
+  onTestStart(test) {
+    const { path } = test
+    console.info(`\n**  Running ${parse(path).name} at ${this.timeNow()}\n`)
+  }
```

Comment:
How would you feel about saving some vertical space with
```suggestion
  onTestStart({ path }) {
    console.info(`\n**  Running ${parse(path).name} at ${this.timeNow()}\n`)
  }
```
while making it clear from the beginning what in `test` we pull out?

Reply:
I like this. But for this function, I prefer to leave original code unchanged, to stick to the Jest `Reporter` interface defined [here](https://github.com/facebook/jest/blob/816789920fe08b2f0245963e91d3a96fe80e1311/packages/jest-reporters/src/types.ts#L25).